### PR TITLE
change logger

### PR DESCRIPTION
**Pull request by maathor**

### CODE REVIEW

File: src/cmd/kubectl.rs [hunk 1]
```diff
@@ -514,9 +514,7 @@ where
             ),
         },
         Err(e) => warn!(
-            "Unable to execute describe on secrets: {}. it may not exist anymore?",
```
Comment (deimosfr) [inline src/cmd/kubectl.rs:517]: Why did you remove this? Wasn't it useful?

File: tests/aws/aws_databases.rs [hunk 2]
```diff
@@ -56,6 +65,12 @@ fn deploy_an_environment_with_3_databases_and_3_apps() {
 fn postgresql_deploy_a_working_development_environment_with_all_options() {
     init();
 
+    let span = span!(
```
Comment (deimosfr) [inline tests/aws/aws_databases.rs:68]: Couldn't it be simplified? Using a function for example?

Comment (maathor) [reply tests/aws/aws_databases.rs:68]: unfortunately, span lives on function they are called

File: src/cmd/kubectl.rs (hunk 1 above)
Comment (maathor) [reply src/cmd/kubectl.rs:517]: nop, IMO

**Review by deimosfr — APPROVED** [review]
